.github/workflows/webhook.yml

```diff
@@ -0,0 +1,104 @@
+name: webhook-workflow
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - operator/webhook/**
+      - .github/workflows/webhook.yml
+  push:
+    branches:
+      - main
+    paths:
+      - operator/**
+      - "!operator/webhook/**"
+      - .github/workflows/operator.yml
+
+env:
+  WORKING_DIR: ./operator/webhook
+  REGISTRY: ghcr.io/octoconsulting
+  IMAGE_NAME: keip/route-webhook
```

Comment:
:question: Same question about syncing to a single source of truth here. Can we get this to track the Makefile `IMAGE_NAME` and `REGISTRY`?

Reply:
Updated to grab registry and image-name from the pom.xml as well.

Reply:
Added getters to Makefile to have source of truth in Makefile.